newIDE/app/src/InstancesEditor/InstancesResizer.js

```diff
@@ -106,7 +107,7 @@ export default class InstancesResizer {
     ));
   }
 
-  _getOrCreateInstanceOriginPosition(instance: gdInitialInstance) {
+  _getOrCreateInstanceOriginPosition(instance: gdInitialInstance): {} {
```

Comment:
This type is wrong, either use the proper type or let Flow infer it
```suggestion
  _getOrCreateInstanceOriginPosition(instance: gdInitialInstance) {
```